### DISCUSSION TITLE
🧨我要榨乾學校(office、Windows通通免費)

### DISCUSSION THREAD

**Discussion by JunTingLin** [1101-GS4719-C/Python-Community-GS4719-C#390] [General]
# 🧨我要~~榨乾~~學校(office、Windows通通免費)

## 前言
最為中央大學的學生，有超多好康你知道嗎❓

這裡來跟大家分享如何授權office跟window，透過學校的KMS認證，連一毛錢都不用花就可以囉😁

## window認證

1. 先去下載window的ISO檔或製作快閃磁碟機
可以參考[Win 11完整安裝過程全記錄！從製做Windows 11安裝隨身碟開始，一步一步分享整個安裝過程！](https://www.youtube.com/watch?v=vRoghRxJ73c&ab_channel=%E5%88%86%E4%BA%AB%E5%AE%B6-%E7%BE%BD)

2. 可以從portal進入KMS認證
![螢幕擷取畫面 2022-03-02 210430](https://user-images.githubusercontent.com/92431095/156369672-df808ec7-ef3d-4c9f-8827-a4c9793562c7.png)

![螢幕擷取畫面 2022-03-02 211142](https://user-images.githubusercontent.com/92431095/156369702-2f32687b-9c52-4489-ae3a-7afec3c70cb7.png)

3. 以系統管理員身分打開CMD
4. 在命令提示字元裡貼上:
```
cscript C:\windows\system32\slmgr.vbs -skms kms-v4.cc.ncu.edu.tw
```

5. 按下「暫時開通」按鈕，開啟 5 分鐘內的時間
6. 在命令提示字元裡貼上:
```
cscript C:\windows\system32\slmgr.vbs -ato  
``` 
7. 看到「產品已啟用成功」，表示認證已完成

## Office2021認證

1. 可以到[學校授權軟體](https://ca.ncu.edu.tw/software/module/download/index.php?cid=2)下載Office2021
![螢幕擷取畫面 2022-03-02 211346](https://user-images.githubusercontent.com/92431095/156369754-411ccbb1-6401-4137-812a-f7007cd30c74.png)

2. 請確認上方有出現「暫時開通」的按鈕(表示已登入)
3. 以「系統管理員身分」開啟命令提示字元
4. 啟動 Office 2016, 2019, 2021 (Windows 64-bit 且 Office 是 32-bit)，請輸入
```
cd "C:\Program Files (x86)\Microsoft Office\Office16\"
```
5. 在命令提示字元裡貼上:
```
cscript OSPP.VBS /sethst:kms-v4.cc.ncu.edu.tw
```

6. 按下「暫時開通」按鈕，開啟 5 分鐘內的時間
7. 在命令提示字元裡貼上:
```
cscript OSPP.VBS /act
```

8. 看到「產品已啟用成功」，表示認證已完成

**Comment by pekoranoinu** [comment]
原來我們有這種福利，謝謝大佬分享。

**Comment by Ben6uo** [comment]
哇!那樣就不用額外的錢來買一堆東西了!!謝謝大佬分享。

**Comment by chupochun** [comment]
謝謝分享這麼好康的福利 如此一來可以省下一筆錢了!!!

**Comment by lx3211** [comment]
推推我現在就是用學校的

**Comment by ThomasMeng0515** [comment]
這個是不是畢業以後就不能用了(?

**Comment by JunTingLin** [comment]
我們學校也有提供強大的伺服器供學生連線使用
可以看看這個喔 [💻你的電腦不夠強嗎?別怕~有VDI #409](https://github.com/NCU-GS4719-Python/Python-Community/discussions/409)
